engine/glfw/include/GL/glfw.h

```diff
@@ -570,6 +570,7 @@ GLFWAPI void glfwUnregisterUIApplicationDelegate(void* delegate);
 GLFWAPI void glfwSetViewType(int view_type);
 GLFWAPI void glfwSetWindowBackgroundColor(unsigned int color);
 GLFWAPI float glfwGetDisplayScaleFactor();
+GLFWAPI int glfwGetMouseLocked();
```

Comment:
The mouse locking state can change internally in glfw so needed to add some way of getting that state.